File: c/leap.c
#include<stdio.h>
void main()
{
int n;
printf("enter the year");
scanf("%d",&n);
if(n%4==0)
{
printf("it is leap year");
}
else if (n%400==0)
{
  printf("is leap year");
}
else if(n%100==0)
{
printf("not leap year");
}
}

File: python1/leap.py
year=(int(input("enter the year  ")))

if (year%4)==0:
    if (year%100)==0:
        if(year%400)==0:
            print("is leap year",year)
        else:
                print("is not lea[ year",year)
    else: 
            print("is leap year",year)

else:
            print("is not leap year",year)            